packages/Tables/TableTitle.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import {
+  StyleProp,
+  StyleSheet,
+  TouchableWithoutFeedback,
+  View,
+  ViewStyle,
+} from 'react-native';
+
+import styled from 'styled-components/native';
+
+type Props = React.ComponentPropsWithRef<typeof TouchableWithoutFeedback> & {
+  children?: React.ReactNode;
+  numberOfLines?: number;
+  onPress?: () => void;
+  style?: StyleProp<ViewStyle>;
+};
+
+const StyledText = styled.Text`
+  width: 100%;
+  height: 24px;
+  line-height: 24px;
+  font-size: 12px;
+  align-items: center;
+  text-align: center;
+`;
+
+function TableTitle(props: Props): React.ReactElement {
+  const { children, onPress, style, numberOfLines, ...rest } = props;
```

Comment:
In this case, I think using `...rest` is a good idea since different types of components are underneath. 
`<TouchableWithoutFeedback>` => `<View>` => `<StyledText>`